modules/openid_auth/app/controllers/openid_auth/v0/mvi_users_controller.rb

```diff
@@ -6,22 +6,30 @@ module OpenidAuth
   module V0
     class MviUsersController < ApplicationController
       skip_before_action :authenticate
-      before_action :check_required_headers
+      before_action :check_required_headers, only: :show
 
       # Allows MVI lookups based on user identity traits (presumably obtained from the upstream
       # identity provider). Returns a 200 response when the MVI lookup succeeds. Raises an error
       # otherwise.
       def show
-        user_identity = save_requested_identity
-        service = MVI::Service.new
-        mvi_response = service.find_profile(user_identity)
-        raise mvi_response.error if mvi_response.error
+        user_identity = build_identity_from_headers
+        process_identity(user_identity)
+      end
```

Comment:
It's probably worth a comment above `def show` explaining that this method is deprecated and will be removed just in case we don't get to removing it right away.

Reply:
Is the SAML proxy the only component that uses this endpoint? If so can we create a ticket so we don't forget to remove the deprecated action and specs after the SAML proxy is updated?